conf/nginx.conf

```diff
@@ -4,12 +4,12 @@ set $allowed_origins "https://${sandbox_domain}";
 set $api_domain "api.__DOMAIN__";
 set $files_domain "files.__DOMAIN__";
 ssl_ecdh_curve secp384r1;
-add_header Strict-Transport-Security "max-age=31536000; includeSubDomains" always;
-add_header X-XSS-Protection "1; mode=block";
-add_header X-Content-Type-Options nosniff;
-add_header Access-Control-Allow-Origin "${allowed_origins}";
-add_header Cross-Origin-Resource-Policy cross-origin;
-add_header Cross-Origin-Embedder-Policy require-corp;
+more_set_headers Strict-Transport-Security "max-age=31536000; includeSubDomains" always;
```

Comment:
more_set_headers "Strict-Transport-Security: max-age=31536000; includeSubDomains; always";

Reply:
I think you need quotation marks and colon

Reply:
thanks yes, I am gonna try in a test VPS then so I won't be surprised if it is all broken...
I will come back to CI after I adapted all to more_set_headers instead of add_header if it works...